Example Apps/Example ObjC/VehicleDataManager.m

```diff
@@ -106,12 +106,12 @@ - (void)unsubscribeToVehicleOdometer {
  *
  *  @param notification A SDLOnVehicleData notification
  */
-- (void)vehicleDataNotification:(SDLRPCNotificationNotification *)notification {
-    if (![notification.notification isKindOfClass:SDLOnVehicleData.class]) {
+- (void)vehicleDataNotification:(SDLRPCMessage *)notification {
+    if (![notification isKindOfClass:SDLOnVehicleData.class]) {
```

Comment:
The class check always fails because the `OnVehicleData` notification is embedded in a `userInfo` object.

```
SDLRPCNotificationNotification 0x600002cd2740 {name = com.sdl.notification.vehicleData; object = <SDLNotificationDispatcher: 0x600002eb10c0>; userInfo = {
    SDLNotificationUserInfoObject = "OnVehicleData (notification)\n{\n    odometer = 123;\n}";
}}
```

Reply:
The Swift example app's `notification` data type is `SDLRPCNotificationNotification` but the Obj-C example app's data type is `SDLRPCMessage`